client/app/lib/styl/changeteammodal.styl

```diff
@@ -0,0 +1,146 @@
+.change-team-modal.kdmodal
+  -webkit-font-smoothing    antialiased
+  -moz-osx-font-smoothing   grayscale
+  rounded                   6px
+
+  .kdmodal-inner
+    margin                  0
+
+    .kdmodal-content
+      margin                0
+      overflow              visible
+      font-family           ProximaNovaFamily
+
+  .close-icon.closeModal
+    r-sprite                app, modal-close-small
+    top                     18px
+    right                   18px
+    z-index                 10
+
+    &:hover
+      r-sprite              app, modal-close-small
```

Comment:
we don't use hyphens on sprite names the sprite builder may randomly fail on them. pls use underscores.